usaspending_api/search/tests/integration/spending_by_category/test_elasticsearch_header.py

```diff
@@ -18,7 +18,7 @@ def test_elasticsearch_headers(client, monkeypatch, elasticsearch_transaction_in
 def elasticsearch_http_header_helper(client, monkeypatch, elasticsearch_transaction_index, endpoint_name):
     logging_statements = []
     monkeypatch.setattr(
-        "usaspending_api.search.v2.views.spending_by_category_views.base_spending_by_category.logger.info",
+        "usaspending_api.search.v2.views.spending_by_category_views.abstract_spending_by_category.logger.info",
```

Comment:
@sethstoudenmier now that I see this, I'm thinking it would be a more concise, more pythonic packaging structure to just put the `abstract` class and any of its concrete implementations into the same module.
That is,
- rename `abstract_spending_by_category.py` to `spending_by_category.py`
- and the others too, e.g. `abstract_spending_by_industry_codes.py` -> `spending_by_industry_codes.py`, `abstract_spending_by_agency.py` -> `spending_by_agency.py`
- and then put the definition for e.g. `class CfdaViewSet(AbstractIndustryCodeViewSet):` into `spending_by_industry_codes.py`

This current structure of having a module (file) per abstract class and concrete impl feels more Java-esque where it requires a file per Class, and I think python code would lean towards bundling them in the same module. 

Is there a reason to separate them? e.g. did code climate complain on length of the abstract file?

Reply:
No real reason to have them broken apart. Codeclimate probably will complain though.